sdcm/utils/version_utils.py

```diff
@@ -39,6 +39,8 @@
 #   - 2019.1.4-0.20191217.b59e92dbd
 
 # gemini version 1.0.1, commit ef7c6f422c78ef6b84a6f3bccf52ea9ec846bba0, date 2019-05-16T09:56:16Z
+from sdcm.utils.decorators import retrying
```

Comment:
Add this import in "sdcm" import group above please:

```
...
from pkg_resources import parse_version
from repodataParser.RepoParser import Parser

from sdcm.utils.common import ParallelObject, DEFAULT_AWS_REGION
from sdcm.utils.decorators import retrying
from sdcm.sct_events.system import ScyllaRepoEvent
...
```

Reply:
it is pycharm magic, fixed.